modules/concept-docs/examples/buckets_and_clusters.java

```diff
@@ -0,0 +1,46 @@
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.ClusterOptions;
+import com.couchbase.client.java.Collection;
+import com.couchbase.client.java.Scope;
+import com.couchbase.client.java.env.ClusterEnvironment;
+import com.couchbase.client.java.manager.bucket.BucketManager;
+import com.couchbase.client.java.manager.bucket.BucketSettings;
+
+public class buckets_and_clusters {
```

Comment:
can you modify the class and file names to use the uppercase syntax like in the topmost file?

Reply:
Will do.